students/zach_thomson/lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,14 @@
+def fizzBuzz():
+    i = 1
+    while i <= 100:
```

Comment:
note that you could have used `range` here which is more appropriate

Reply:
So it'd be more appropriate to have my while statement look like:

while i in range(101)